### ISSUE TITLE
Hashed course_uuid and Instructor ID values change on each iteration

### ISSUE THREAD

**Issue by samjett247** [stev-ou/stev-api#32]
Hash functions change over consecutive runs, making it very difficult to keep frontend in tune with back. Example here:
**Merge PR1 on backend ** 'Bill Smith' -> 43425245252
**Merge PR2 on backend** 'Bill Smith' -> 98732789472

Ideally, we would use a hash function that doesn't change each time its used; That way we can post hyperlinks to instructors that wont change with each iteration.

Thoughts on a hash function that doesn't change with time @jlovoi @schuermannator ? Would be a super small change in `data_loader.py`, could probably be included with the graphQL stuff.

**Comment by zachschuermann** [comment]
Hashing by definition has to be deterministic... this is really weird. Maybe we need to look into the specific input/output for each of those cases? Could the input have somehow changed @samjett247?  
  
I see the hashing here:
```python3
df['Instructor ID'] = (df['Instructor First Name']+df['Instructor Last Name']).apply(str).apply(hash).astype('int32').abs()
df['course_uuid'] = (df['Subject Code']+df['Course Number'].apply(str)+df['Section Title'].apply(lambda x: x[:-4])).apply(str).apply(hash).astype('int32').abs().apply(str)
```
but I tested even with the pandas df and applying types and still stayed deterministic (as far as my quick tests went)  
  
For reference:  
```python3
>>> df = pd.DataFrame(["thisisatest"])
>>> df[0].apply(str).apply(hash).astype('int32').abs()
39314118
```

**Comment by samjett247** [comment]
Peep this screenshot; Try repeating on yours. I think `hash()` in python is deterministic inside of a namespace/while using the same instance of the interpreter, but between two python instances its not deterministic. I'll find something inside of hashlib which satisfies our problems; Should be a quick drop-in switch to fix. Look for PR soon.

<img width="1077" alt="Screen Shot 2019-09-05 at 9 31 49 PM" src="https://user-images.githubusercontent.com/30481194/64401283-1ec36500-d025-11e9-8cdc-32cf17d8a5d9.png">

**Comment by samjett247** [comment]
Yup confirmed. New apply function using sha224 was implemented in #33.  confirmed to be deterministic from unit tests, and confirmed to work in integration by rerunning data loader 😄 Issue solved I think.

**Comment by zachschuermann** [comment]
Nice. I guess it was a bit hacky using python's built-in `hash` this is great thanks
